doc/source/release-notes/Percona-Server-5.6.40-84.0.rst

```diff
@@ -0,0 +1,158 @@
+
+* A ``key_block_size`` value was set automatically by the Improved MEMORY
+  Storage Engine, which resulted in warnings when changing the engine type to
+  |InnoDB|, and constantly growing ``key_block_size`` during alter operations.
+  Bugs fixed :psbug:`3936`, :psbug:`3940`, and :psbug:`3943`
+
+* |Percona Server| Debian packages description included reference to
+  ``/etc/mysql/my.cnf`` file, which is not actually present in these packages.
+  Bug fixed :psbug:`2046`.
+
+* Fixes were introduced to remove GCC 8 compilation warnings for the
+  |Percona Server| build, retaining compatibility with old compiler versions,
+  including GCC 4.4. Bugs fixed :psbug:`3950` and :psbug:`4471`.
+
+* A typo in ``plugin.cmake`` file prevented to compile plugins statically
+  into the server. Bug fixed :psbug:`3871` (upstream :mysqlbug:`89766`).
+
+* ``-DWITH_NUMA=ON`` build option was silently ignored by CMake when
+  NUMA development package was not installed, instead of exiting by error.
+  Bug fixed :psbug:`4487`.
+
+* Variables :variable:`innodb_buffer_pool_populate` and :variable:`numa_interleave` mapped to the upstream `innodb_numa_interleave <http://dev.mysql.com/doc/refman/5.6/en/innodb-parameters.html#sysvar_innodb_numa_interleave>`_ variable in :rn:`5.6.27-75.0` were reverted to their original implementation due to upstream variant being less effective at memory pages fault because of not using ``MAP_POPULATE`` when NUMA interleave is turned on. Bug fixed  :psbug:`3967`.
+
+* :variable:`audit_log_include_accounts` variable did not take effect if
+  placed in ``my.cnf`` configuration file, while still working as intended if
+  set dynamically. Bug fixed :psbug:`3867`.
+
+* Synchronization between between :variable:`innodb_kill_idle_transaction` and
+  :variable:`kill_idle_transaction` system variables was broken because of the
+  regression in |Percona Server| :rn:`5.6.40-83.2`. Bug fixed :psbug:`3955`.
+
+* Executing the ``SHOW GLOBAL STATUS`` expression could cause "data drift" on
+  global status variables in case of a query rollback: the variable, being by
+  its nature a counter and allowing only an increase, could return to its
+  previous value. Bug fixed :psbug:`3951` (upstream :mysqlbug:`90351`).
+
+* ``ALTER TABLE ... COMMENT = ...`` statement caused |TokuDB| to rebuild the
+  whole table, which is not needed, as only FRM metadata should be changed. The
+  fix was provided as a contribution by `Fungo Wang <https://github.com/fungo>`_.
+  Bugs fixed :psbug:`4280` and :psbug:`4292`.
+
+* A number of |Percona Server| 8.0 |TokuDB| fixes have been backported to
+  |Percona Server| 5.6 in preparation for using MySQL 8.0. Bugs fixed 
+  :psbug:`4379`, :psbug:`4380`, :psbug:`4387`, :psbug:`4378`, :psbug:`4383`,
+  :psbug:`4384`, :psbug:`4386`,  :psbug:`4382`, :psbug:`4391`,
+  :psbug:`4390`, :psbug:`4392`, and :tdbbug:`4381`.
+
+TokuDB Changes and fixes
+========================
+
+* Two new variables,   :variable:`tokudb_enable_fast_update` and
```

Comment:
This is much better.  Since this is default to disabled, however, I am wondering if we can also note why that is the case.  Such as, if there's a specific known downside that prevented us from enabling fast updates by default?  I am seeing this as a question that may be asked by the end-user, since they will say "Of course I want my updates to be fast, but why isn't this default?"

Reply:
Primarily because this is a slight behavior change and thus we don't enable behavior changes b default in mid-series releases. The feature is really only beta quality and it is entirely possible that there are edge cases where its use could introduce index inconsistencies. For those reasons, we do not enable it by default.